### ISSUE TITLE
Error: Call to a member function getAcceptsJson() on string

### ISSUE THREAD

**Issue by denisyilmaz** [barrelstrength/craft-sprout-reports#98]
After installation when trying to access the settings page the CP returns the attached error. Can't figure out why this is happening. Reports as are working.

### Steps to reproduce

1. Install Sprout Resports
2. Click on Sprout Reports Icon on Settings page / same for `SideNav` > `Sprout Reports` > `Data Sources` and `Settings` entries

### Additional info

- Craft version: 3.5.7
- nginx/1.17.5

```bash
Error: Call to a member function getAcceptsJson() on string in /var/www/html/vendor/craftcms/cms/src/web/Controller.php:183
Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('edit-settings', Array)
#1 /var/www/html/vendor/craftcms/cms/src/web/Application.php(280): yii\base\Module->runAction('sprout/settings...', Array)
#2 /var/www/html/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('sprout/settings...', Array)
#3 /var/www/html/vendor/craftcms/cms/src/web/Application.php(265): yii\web\Application->handleRequest(Object(craft\web\Request))
#4 /var/www/html/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#5 /var/www/html/web/index.php(21): yii\base\Application->run()
#6 {main}
```

**Comment by BenParizek** [comment]
@denisyilmaz This is due to a change in behavior in Craft 3.5

I'm currently working on a new, major release with 3.5 support that will address this and have not tested the current version of the plugins on Craft 3.5

If you'd like to address this issue temporarily, you can make the following changes, but there might be other issues that come up too until I have a chance to get the latest major release out.

To fix this issue.

In barrelstrength/sprout-base
https://github.com/barrelstrength/craft-sprout-base/commit/f3aa42c438a4287d4c268aa0fb59947a6ea06ba5

In barrelstrength/sprout-reports
https://github.com/barrelstrength/craft-sprout-reports/commit/ddcf4119fb6c8d16f4e67541690f886c04346d6c

**Comment by BenParizek** [comment]
This has been addressed in v1.5.7
